matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+            type: Gauge
+            description: Number of work items currently queued for processing by the thread pool. Only available for .NET 6 or higher.
+          - metric: process.runtime.dotnet.jit.il_compiled.size
+            type: Cumulative counter
+            description: Bytes of intermediate language that have been compiled since the process started. Only available for .NET 6 or higher.
+          - metric: process.runtime.dotnet.jit.methods_compiled.count
+            type: Cumulative counter
+            description: Number of times the JIT compiler compiled a method since the process started. Only available for .NET 6 or higher.
+          - metric: process.runtime.dotnet.jit.compilation_time
+            type: Cumulative counter
+            description: Amount of time the compiler spent compiling methods since the process started. Only available for .NET 6 or higher.
+          - metric: process.runtime.dotnet.timer.count
+            type: Gauge
+            description: Number of timer instances currently active. Only available for .NET 6 or higher.
+          - metric: process.runtime.dotnet.assemblies.count
+            type: Gauge
+            description: Number of .NET assemblies that are currently loaded.
+          - metric: process.runtime.dotnet.exceptions.count
+            type: Cumulative counter
+            description: Count of exceptions thrown in managed code since the observation started.
+  - keys:
+      - PROCESS
+    library: Process
+    version: any
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Process
+        version: 0.5.0-beta.3
+        stability: beta
+    signals:
+      - metrics:
+          - metric: process.memory.usage
+            type: Gauge
+            description: The amount of physical memory allocated for this process.
+          - metric: process.memory.virtual
+            type: Gauge
+            description: The amount of committed virtual memory for this process.
+          - metric: process.cpu.time
+            type: Cumulative counter
+            description: Total CPU seconds broken down by different states, such as user and system.
+          - metric: process.cpu.count
+            type: Gauge
+            description: Total CPU seconds broken down by different states, such as user and system.
+          - metric: process.threads
+            type: Gauge
+            description: Process threads count.
+  - keys:
+      - ILOGGER
+    library: Microsoft.Extensions.Logging
+    version: 6.0.0 and higher
+    comment: Not supported on .NET Framework
+    stability: beta
+    supportabilityLevel: community
+    signals:
+      - logs: 
+resourceDetectors:
+  - key: AZUREAPPSERVICE
+    description: Azure App Service detector.
+    attributes:
+      - azure.app.service.stamp
```

Comment:
```suggestion
      - key: azure.app.service.stamp
```

to avoid potential breakin change if we would e.g. need a description

Reply:
I will call it is, same as in sem-conv https://github.com/open-telemetry/semantic-conventions/blob/b127e121b9b458e83a0fee8a32f7eb6721912951/model/resource/process.yaml

Reply:
350036e5dd0e03770d22f38e8f818dd60bd77b57